rfc/async-first-team.md

```diff
@@ -0,0 +1,92 @@
+
+Operating numerous services, our team is dispersed across multiple countries, leading to potential timezone variances of up to 10 hours. This global presence fosters around-the-clock productivity and a diverse perspective within our collaborative endeavors."
+
+## Async vs sync teams
+
+We recognize that each methodology offers its own set of advantages and drawbacks. Various teams may excel under different methodologies, influenced by factors such as team seniority, physical proximity, time zone disparities, and individual preferences.
+
+### Advantages of Async Team Communication
+
+1. Flexibility: Async communication allows team members to engage with work-related discussions and tasks at their convenience, accommodating diverse schedules and time zones.
+2. Reduced Interruptions: By minimizing synchronous meetings and real-time communication channels, async methodologies help mitigate interruptions, allowing individuals to focus on deep work and productivity.
+3. Increased Thoughtfulness: With async communication, individuals have more time to reflect on their responses and contributions, leading to more considered and well-articulated ideas. Moreover, by allowing participants to engage at their own pace and reducing the pressure of immediate responses, async methodologies create a more supportive environment where individuals with challenges such as social anxiety can contribute comfortably and constructively to team discussions.
+4. Documentation: Asynchronous communication encourages the documentation of discussions and decisions, fostering transparency and ensuring that information is accessible to all team members, as well as a ramp up for new members joining the team.
+5. Expanded Hiring Opportunities: Embracing async communication widens the talent pool by accommodating individuals from diverse backgrounds and geographic locations. This inclusivity fosters a richer variety of perspectives, experiences, and skills within the team, ultimately enhancing innovation and problem-solving capabilities.
+6. Streamlined Onboarding Process: Async communication promotes thorough documentation of discussions, decisions, and project information, providing valuable resources for new team members during the onboarding process. This accessibility to historical context and archived conversations facilitates a smoother transition and accelerates the integration of newcomers into the team.
+7. Mitigated Knowledge Loss: With comprehensive documentation and asynchronous communication norms in place, the risk of losing critical knowledge during employee offboarding is significantly reduced. Team members can easily access archived information and refer to previous discussions, ensuring continuity of work and minimizing disruptions caused by personnel changes.
```

Comment:
Also value this one, especially to be able to retain context in a ever-evolving system whose individual parts no one is going to be able to consistently be an expert in

Reply:
Not sure it's worth calling out directly, but choosing git repos (like rfc, spec, etc) over e.g. slack does foster better information organization and retention than e.g. slack threads and alike.